arches/app/models/models.py

```diff
@@ -1280,6 +1252,118 @@ class Meta:
         permissions = (("no_access_to_resourceinstance", "No Access"),)
```

Comment:
Good question. @robgaston what's the right move here?

Reply:
Yeah, I would say we should do that.  it wouldn't make sense as things currently exist to ever design a lifecycle against a branch as you never collect data against a branch and it would just be lost on append to a resource model (i presume) so I would say we should add this constraint.